.github/workflows/ci_test-full.yml

```diff
@@ -127,6 +120,7 @@ jobs:
           pip install --no-cache-dir -r requirements/horovod.txt
         fi
         horovodrun --check-build
+        python -c "import horovod.torch; horovod.torch.nccl_built()"
```

Comment:
Might as well do it in the newly added file with the import

```suggestion
```

Reply:
here we assume only this `horovod` strategy so don't think it needs to make a special file for it...